config/config.php

```diff
@@ -20,6 +20,11 @@
     \Zend\Expressive\ConfigProvider::class,
     \Zend\Expressive\Router\ConfigProvider::class,
 
+    // Swoole config to overwrite some services (if installed)
+    class_exists(\Zend\Expressive\Swoole\ConfigProvider::class)
+        ? \Zend\Expressive\Swoole\ConfigProvider::class
+        : null,
```

Comment:
This doesn't seem to work: `Zend\ConfigAggregator\InvalidConfigProviderException: Cannot read config from NULL - config provider must be callable.`